bukkit/src/main/java/com/github/games647/fastlogin/bukkit/listener/protocollib/LoginSkinApplyListener.java

```diff
@@ -20,7 +20,7 @@ public LoginSkinApplyListener(FastLoginBukkit plugin) {
     }
 
     @EventHandler(priority = EventPriority.LOW)
-    public void onPlayerLogin(PlayerLoginEvent loginEvent) {
+    public void onPlayerLogin(PlayerJoinEvent loginEvent) {
```

Comment:
So player was null?

How can that be null? Do had any plugins besides ProtocolLib on your server? I tested it recently and it worked.